database/migrations/2022_11_07_171403_add_config_descriptions.php

```diff
@@ -0,0 +1,116 @@
+		DB::table('configs')->where('key', 'thumb_2x')->update(['description' => 'Enable 2x size of square thumbs']);
+		DB::table('configs')->where('key', 'small_2x')->update(['description' => 'Enable 2x size of small thumbs']);
+		DB::table('configs')->where('key', 'medium_2x')->update(['description' => 'Enable 2x size of medium pictures']);
+		DB::table('configs')->where('key', 'site_title')->update(['description' => 'Website title']);
+		DB::table('configs')->where('key', 'site_copyright_enable')->update(['description' => 'Enable copyright notice at the bottom']);
+		DB::table('configs')->where('key', 'site_copyright_begin')->update(['description' => 'Initial year of copyright']);
+		DB::table('configs')->where('key', 'site_copyright_end')->update(['description' => 'Last year of copyright']);
+		DB::table('configs')->where('key', 'api_key')->update(['description' => 'Deprecated']);
+		DB::table('configs')->where('key', 'allow_online_git_pull')->update(['description' => 'Allow git pull via web interface']);
+		DB::table('configs')->where('key', 'force_migration_in_production')->update(['description' => 'Force migration even if app is production mode']);
+		DB::table('configs')->where('key', 'additional_footer_text')->update(['description' => 'Extra text at the bottom of the page']);
+		DB::table('configs')->where('key', 'display_social_in_gallery')->update(['description' => 'Display social links at the bottom of the gallery']);
+		DB::table('configs')->where('key', 'public_search')->update(['description' => 'Allows anonymous user to use the Search bar']);
+		DB::table('configs')->where('key', 'gen_demo_js')->update(['description' => 'Enable generation of JS responses for demo purposes']);
+		DB::table('configs')->where('key', 'hide_version_number')->update(['description' => 'Hide current version number']);
+		DB::table('configs')->where('key', 'public_recent')->update(['description' => 'Make Recent smart album accessible to anonymous users']);
+		DB::table('configs')->where('key', 'recent_age')->update(['description' => 'Maximum age of pictures in recent in days']);
+		DB::table('configs')->where('key', 'public_starred')->update(['description' => 'Make Starred smart album accessible to anonymous users']);
+		DB::table('configs')->where('key', 'SL_enable')->update(['description' => 'Symbolic Link protection enabled']);
+		DB::table('configs')->where('key', 'SL_for_admin')->update(['description' => 'Also enable symbolic links on logged admin users']);
+		DB::table('configs')->where('key', 'SL_life_time_days')->update(['description' => 'Maximum life time for symbolic link']);
+		DB::table('configs')->where('key', 'photos_wraparound')->update(['description' => 'Once reaching last picture of an album, loop back to the start']);
+		DB::table('configs')->where('key', 'raw_formats')->update(['description' => 'Allowed extra format, will not be processed']);
```

Comment:
```suggestion
		DB::table('configs')->where('key', 'raw_formats')->update(['description' => 'Allowed extra formats, will not be processed']);
```
Are RAWs not processed? I thought we try to generate JPEG previews if ImageMagick is enabled?